src/client/reducers/audioReducer.js

```diff
@@ -0,0 +1,64 @@
+import { CREATE_AUDIO_DATA, PLAY_PAUSE_SOUND_FROM_FILE, CREATE_STREAME_DATA, START_MUTE_STREAME_AUDIO, MERGE_CANVAS_WIDTH } from '../actions/types';
+
+var context = new (window.AudioContext || window.webkitAudioContext)();    
+var analyser = context.createAnalyser();
+
+
+const initialState={  
```

Comment:
Please use camel cases for properties names

Reply:
renamed